### PR TITLE
Switch ocean story map engine to openlayers 3

### PR DESCRIPTION
**Pull request by edrex**

### CODE REVIEW

File: marco_site/static/bower.json [hunk 1]
```diff
@@ -14,10 +14,7 @@
     "jqueryui-touch-punch": "*",
     "bootstrap3-typeahead": "~3.1.0",
     "jquery-ui": "~1.11.2",
-    "leaflet": "~0.7.3",
-    "leaflet-plugins": "git@github.com:shramov/leaflet-plugins.git#v1.2.0",
     "lodash": "~2.4.1",
-    "Proj4Leaflet": "https://github.com/DISID/Proj4Leaflet.git#geojson-empty-constructor",
-    "esri-leaflet": "~1.0.0-rc.4"
+    "ol3-unofficial": "~3.0.3"
```
Comment (rcranley) [inline marco_site/static/bower.json]: Is this because OL3 doesn't have an official bower package?

Comment (edrex) [inline marco_site/static/bower.json]: There is an "ol3" entry in the bower registry which points to the main ol3 github repo, but it's giant. The [unofficial one](https://github.com/nd0ut/ol3-bower) is just 3 files

**Comment by rcranley** [comment]
This looks good.
An update to the ReadMe is needed too, to reflect no longer using Leaflet...

**Comment by edrex** [comment]
updated readme

**Comment by sfletche** [comment]
@edrex - finally looking at this commit...a couple of comments...

the Open Street Map source can be pulled like this
`new ol.source.OSM(),` rather than `new ol.source.MapQuest({layer: 'osm'}),`

my second comment may be nit-picky...I'm not a big fan of single letter variables (other than counters), what do you think about replacing the function parameters 'l' with 'layer'?

**Pull request merged**